src/keycloak/keycloak_openid.py

```diff
@@ -174,7 +174,7 @@ def well_known(self):
 
         return raise_error_from_response(data_raw, KeycloakGetError)
 
-    def auth_url(self, redirect_uri):
+    def auth_url(self, redirect_uri, scope="email", state=""):
```

Comment:
Could I ask you to add these into the docstring? :)

Reply:
> Could I ask you to add these into the docstring? :)

done!

Reply:
By docstring I meant the method's docstring, not the README file :) i.e. something like

```python
def auth_url(self, redirect_uri, scope="email", state=""):
    """
    Get authorization URL endpoint.

    :param redirect_uri: Redirect url
    :type redirect_uri: str
    :param scope: Scope
    :type: scope: str
    :param state: State
    :type: str
    :returns: Authorization URL
    :rtype: str
    """"
    ...
```

Reply:
Thank you for reminding me.
But I'm in bed now.I will deal with it after I get up.

Reply:
@ryshoooo 
 Everything Done！
 Check pls！